mmd_tools/panels/tool.py

```diff
@@ -31,6 +31,7 @@ def draw(self, context):
         col = layout.column(align=True)
         col.operator(operators.material.ConvertMaterialsForCycles.bl_idname, text='Convert Materials For Cycles')
         col.operator('mmd_tools.separate_by_materials', text='Separate By Materials')
+        col.operator(operators.misc.JoinMeshes.bl_idname)
 
```

Comment:
As you should have noticed, @sugiany have the practice of referencing the bl_idname, probably this is to allow the possibility of changing the name easily. Anyway I just follower his/her practice.